src/main/java/roomescape/presentation/exception/GlobalExceptionHandler.java

```diff
@@ -69,6 +70,17 @@ public ResponseEntity<ErrorResponse> handlePastReservationException(PastReservat
         return ResponseEntity.badRequest().body(errorResponse);
     }
 
+    @ExceptionHandler(PaymentException.class)
+    public ResponseEntity<ErrorResponse> handlePaymentException(PaymentException e) {
+        String message = e.getMessage();
+        HttpStatus status = e.getStatus();
+        if (status.is5xxServerError()) {
+            handleAllExceptions(e);
```

Comment:
```suggestion
            return handleAllExceptions(e);
```
return 되어야하는게 아닐까요~?

Reply:
오.. 그렇네요!
[토스 에러 코드](https://docs.tosspayments.com/reference/error-codes#%EC%BD%94%EC%96%B4-api-%EB%B3%84-%EC%97%90%EB%9F%AC)에서 제공되는 목록을 확인해보니 401 예외를 제외하고 500 예외도 사용자에게 그대로 메시지를 내려보내주어도 괜찮을 것 같아서 401 예외만 분기처리해주었습니다.
eb1e8c61 에 반영했어요!